app/state.go

```diff
@@ -0,0 +1,247 @@
+package app
```

Comment:
it'd be good to include a copyright header with licensing information if this is copied from Cosmos SDK

Reply:
Done